src/arche/rules/result.py

```diff
@@ -261,3 +267,39 @@ def make_annotations(stat: pd.Series) -> List[Dict]:
                 )
             )
         return annotations
+
+    @staticmethod
+    def build_box_subplots(stat: pd.DataFrame) -> go.Figure:
+        """Create a figure with box subplots showing fields coverages for jobs.
+
+        Args:
+            stat: a dataframe with field coverages
+
+        Returns:
+            A figure with box subplots
+        """
+        stat = stat.drop(columns=["std", "mean", "last std"])
+        traces = [
+            go.Box(
+                y=row[1],
+                name=row[0],
+                boxpoints="all",
+                jitter=0.3,
+                boxmean="sd",
+                hoverinfo="y",
+            )
+            for row in stat.iterrows()
+        ]
+        cols = 4
+        rows = math.ceil(len(stat) / cols)
+        fig = make_subplots(rows=rows, cols=cols)
+        x = 0
+        for i, j in itertools.product(range(1, rows + 1), range(1, cols + 1)):
+            if x == len(traces):
```

Comment:
If I'm not wrong, an alternative would be `x = i * cols + j`

Reply:
You want to simplify? I thought about it. 
`x = i * cols + j` doesn't make sense since x is the trace (a piece of data for 1 plot) index, i is the row index, j for columns
so `0 != 1 * 4 + 1`
I want to keep the number of columns fixed (4 seems the best number).

With `make_subplots` [api](https://plot.ly/python/subplots/) we need to supply row\column index, so we need to have two iterators over them.
Then we need to have an independent iterator over `traces` to place all graphs in these positions, and product with break is the best I could come.

Reply:
Sorry, missed that it started in 1 and not in 0.
If it starts in zero then it would work.